docs/reference/python/v0/collection/collection-query-pagingfrom.md

```diff
@@ -0,0 +1,34 @@
+---
+title: collection.query.pagingFrom()
+description: Set a point to resume a query from.
+---
+
+Set a point to resume a query from, this is required when continuing a paginated query response.
+
+```python
+from nitric.resources import collection
+
+query = docs.collection("profiles").allow('reading').query().limit(1000)
+
+# Fetch first page
+results = await query.fetch()
+
+# Fetch next page
+if results.paging_token:
+    results = await query.page_from(results.paging_token).fetch()
+```
+
+## Parameters
+
+---
+
+**pagingToken** required `unknown`
```

Comment:
Should `unknown` instead be `Any` or `object`?

Reply:
Not sure for python, in typescript `unknown` is actually correct

Reply:
Yeah python doesn't have an unknown type. I think the closest in python is `object`